manifest.ts

```diff
@@ -0,0 +1,32 @@
+import {AppBindingLocations} from 'mattermost-redux/constants/apps';
+
+import {AppID} from './src/utils';
+
+type RequestedPermissions = 'act_as_user' | 'act_as_bot'
+
+export type Manifest = {
```

Comment:
Why is this file needed given that we have `manifest.json`?

Reply:
put into catch-all NITs ticket https://mattermost.atlassian.net/browse/MM-33787